<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalWin</class>
 <widget class="QMainWindow" name="TerminalWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminal easier</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>210</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>210</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>210</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>210</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>450</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>530</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_0">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>210</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(42, 31, 95);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton_0">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="terminal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(229, 230, 244);background-color: rgb(53, 43, 65);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="markdown">
     <string/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="incomingCnt">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(216, 220, 240);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="incomingCntLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>281</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Incoming bytes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>230</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>230</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>230</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>230</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_0">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>230</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>570</y>
      <width>230</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>210</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="plainText" stdset="0">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>570</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveCmdButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Save
Cmds</string>
    </property>
   </widget>
   <zorder>terminal</zorder>
   <zorder>lineEdit_1</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>sendButton_1</zorder>
   <zorder>sendButton_2</zorder>
   <zorder>sendButton_3</zorder>
   <zorder>sendButton_4</zorder>
   <zorder>clearButton</zorder>
   <zorder>lineEdit_0</zorder>
   <zorder>sendButton_0</zorder>
   <zorder>openButton</zorder>
   <zorder>comboBox</zorder>
   <zorder>incomingCnt</zorder>
   <zorder>incomingCntLabel</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>label_1</zorder>
   <zorder>label_0</zorder>
   <zorder>label_5</zorder>
   <zorder>lineEdit_5</zorder>
   <zorder>sendButton_5</zorder>
   <zorder>saveCmdButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSend">
    <property name="title">
     <string>Configure</string>
    </property>
    <widget class="QMenu" name="menuDisplay">
     <property name="title">
      <string>Display</string>
     </property>
     <addaction name="actionSyncCharacters"/>
    </widget>
    <addaction name="actionPort"/>
    <addaction name="menuDisplay"/>
   </widget>
   <addaction name="menuSend"/>
  </widget>
  <action name="menuConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionPort">
   <property name="text">
    <string>Port</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSyncCharacters">
   <property name="text">
    <string>Sync characters</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>label_0</tabstop>
  <tabstop>lineEdit_0</tabstop>
  <tabstop>sendButton_0</tabstop>
  <tabstop>label_1</tabstop>
  <tabstop>lineEdit_1</tabstop>
  <tabstop>sendButton_1</tabstop>
  <tabstop>label_2</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>sendButton_2</tabstop>
  <tabstop>label_3</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>sendButton_3</tabstop>
  <tabstop>label_4</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>sendButton_4</tabstop>
  <tabstop>label_5</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>sendButton_5</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>saveCmdButton</tabstop>
  <tabstop>terminal</tabstop>
  <tabstop>incomingCnt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
